src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -27,7 +27,7 @@ public Movie add(Movie movie) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't insert movie " + movie, e);
+            throw new DataProcessingException("Can't insert movie: " + movie, e);
```

Comment:
The exception handling here is appropriate, as it rolls back the transaction if an error occurs and wraps the exception in a `DataProcessingException`. This provides a clear message about the failure and includes the original exception for debugging.